script/Arcueid/Day 3/03_00_ARC03_1_2.txt

```diff
@@ -135,12 +135,13 @@ Ah, since you get along so well, would you like to be her valet? I would not min
 [sha:1c79225e0fa222e8320b30c53d9ea6dbca35957f]{
 -- Page 53, Offset 4080.
 -- この妹の<棘|パワー>がすごい。
-My sister's <power|b i t i n g  w o r d s     > is truly formidable.
+//don't love this but don't love the ruby either
+My sister's power――her scathing remarks――is truly formidable.
```

Comment:
this is pretty stilted. If you're so inclined you can just turnn "power" into "scathing remarks" or whatever. just reflow the sentence if you do